### PR TITLE
O/price data on events

### PR DESCRIPTION
**Pull request by olegakbarov**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zoralabs/nft-hooks",
-  "version": "1.0.29",
+  "version": "1.0.30",
```
Comment (olegakbarov) [inline package.json:3]: oldie

File: src/types/NFTInterface.ts [hunk 2]
```diff
@@ -207,6 +210,11 @@ type MarketFixedPriceEvent = SharedMarketEventData & {
   side: FIXED_SIDE_TYPES;
   buyer?: ETHAddress;
   seller?: ETHAddress;
+  // TODO: make price data non-optional
+  usd?: CurrencyAmount;
```
Comment (iainnash) [inline src/types/NFTInterface.ts:214]: Perhaps:
```
price: {
  price: {}
  usdcPrice: {}
  nativePrice: {}
}
```

as opposed to this to make what the different types are listed correspond to

Comment (iainnash) [reply src/types/NFTInterface.ts:214]: For optionality, perhaps price? root object is optional. price within is required and usdc/native prices are optional.

Comment (iainnash) [reply src/types/NFTInterface.ts:214]:
```
price?.amount
price?.symbol
price?.usdcPrice?.symbol
price?.nativePrice?.symbol
```

eg: `5 FWB`

<PriceDisplayComponent price={event.price} />

// !! TYPEERROR
<PriceDisplayCompoennt price={event.amount} currency={event.currency} />

File: src/types/NFTInterface.ts [hunk 3]
```diff
@@ -80,6 +80,21 @@ export type CurrencyValue = {
   address: ETHAddress;
 };
 
+export type MarketInfo = {
+  raw: any;
+  amount?: CurrencyValue;
```
Comment (iainnash) [inline src/types/NFTInterface.ts:85]: lgtm

**Review by iainnash — APPROVED** [review]

**Pull request merged**